server_admin/topics/clients/saml/proc-creating-saml-client.adoc

```diff
@@ -84,7 +84,9 @@ This option is used when {project_name} server and adapter provide the IDP and S
 
 *Sign Assertions*:: The assertion is signed and embedded in the SAML XML Auth response.
 
-*Signature Algorithm*:: The algorithm used in signing SAML documents.
+*Signature Algorithm*:: The algorithm used in signing SAML documents. Note that `SHA1` based algorithms are deprecated and can be removed in the future.
+It is recommended to stick to some more secure algorithm instead of `*_SHA1`. Also with `*_SHA1` algorithms verifying signatures
+will not work if the SAML client runs on Java 17 or later.
```

Comment:
```suggestion
We recommend the use of some more secure algorithm instead of `*_SHA1`. Also, with `*_SHA1` algorithms, verifying signatures
do not work if the SAML client runs on Java 17 or higher.
```